src/content/advanced/ingress/multi-nginx-ic/index.md

```diff
@@ -113,6 +113,20 @@ For NGINX IC running on on-prem (KVM) workload clusters there's no out-of-the-bo
 
 More information on this topic can be found in document [Services of type LoadBalancer]({{< relref "/content/advanced/ingress/service-type-loadbalancer/index.md" >}}).
 
+On AWS and Azure, it is also possible to only install a single Nginx Ingress Controller App and to delegate both internal and external traffic to it. Here is a minimal working example on how to achieve this goal.
+
+```yaml
+controller:
+  service:
+    public: true  # default value
+    subdomain: "ingress"  # default value
+    internal: 
+      enabled: true  # default is `false`
+      subdomain: "ingress-internal"  # default value
+```
+
+In other words, it is sufficient to set `controller.service.internal.enabled` to `true` to create to services of type `LoadBalancer`: one for public traffic and one for private one.
```

Comment:
Same here: The specific service type (either `LoadBalancer` or `NodePort`) doesn't matter. The only difference is the way the platform implements the service. In cloud environments you probably go for `LoadBalancer` and let the cloud platform allocate an IP and implement the load balancer, in on-prem environments one would need to create a load balancer pointing to the allocated node ports.

I think even some on-prem platforms already support `LoadBalancer` services, especially when using CAPI. So we could or should just omit the differentiation between `LoadBalancer` and `NodePort` somehow.